app/Services/Families/FamilyTreeService.php

```diff
@@ -110,7 +113,7 @@ protected function prepareEntities(): void
             if ($k !== 'entity_id') {
                 return;
             }
-            if ($v === 0 || in_array($v, $this->configEntityIds)) {
+            if ($v === 0 || in_array($v, $this->configEntityIds) || $v === null) {
```

Comment:
Why a check on `null`? If it's needed, we can combine `0` and `null` with `empty($v)`